CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [1.3.3] - 14.09.2021
+- Support for `KnowledgePanel`s
```

Comment:
We currently just have the models for it, let's announce the KnowledgePanels when we also have the code to use it.

Reply:
OK, I've removed that part